doc/guides/diagnostic-tooling-support-tiers.md

```diff
@@ -0,0 +1,115 @@
+# Diagnostic Tooling Support Tiers
+
+Diagnostic tooling is important to the consumers of Node.js. It is used both
+in development and in production in order to investigate problems.  The failure
+of one of these tools may be as big a problem for an end user as a bug within
+the runtime itself.
+
+The Node.js project has assessed the tools and the APIs which support those
+tools. Each of the tools and APIs has been put them into one of
+the following tiers.  
+
+* Tier 1 - Mmust always be working for all
+  Current and LTS Node.js releases. A release will not be shipped if the test
+  suite for the tool/API is not green. To be considered for inclusion
+  in this tier it must have a good test suite and that test suite and a job
+  must exist in the Node.js CI so that it can be run as part of the release
+  process.  No commit to master should break this tool/API if the next
+  release is within 1 month.
+  
+* Tier 2 - Must be working for all
+  LTS releases. An LTS release will not be shipped if the test
+  suite for the tool/API is not green. To be considered for inclusion
+  in this tier it must have a good test suite and that test suite and a job
+  must exist in the Node.js CI so that it can be run as part of the release
+  process. 
+  
+ * Tier 3 - If possible its test suite
+   will be run at least nightly in the Node.js CI and issues opened for
+   failures.  Does not block shipping a release.
+   
+ * Tier 4 - Does not block shipping a release.
+    
+ * Unclassified - tool/API is new or does not have the required testing in the 
+   Node.js CI in order to qualify for a higher tier.
+   
+The choice of which tier a particular tool will be assigned to, will be a
+collaborative decision between Diagnostics WG and Release WG. Some of the
+criteria considered might be:
+
+* If the tool fits into a key cateogry as listed below.
+* Whether the tool is actively used by the community.
+* The Availability of alternatives.
+* Impact to the overall ecosystem if the tool is not working.
+* The availability of reliable test suite that can be integrated into our CI.
+* The availability of maintainer or community collaborator who will help
+  resolve issues when there are CI failures.
+* If the tool is maintained the Node.js foundation GitHub organization.
+ 
+The current categories of tools/APIs that fall under these Tiers are:
+
+* FDDC (F) - First failure data capture, Easy to consume initial diagnostic
+  information.
+* Tracing (T) - use of logging to provide information about execution flow.
+* Memory (M) - tools that provide additional information about memory 
+  used in the Heap or by native code.
+* Profiling (P) - tools that provide additional information about where
+  cpu cycles are being spent.
+* AsyncFlow (A) - tools that provide additional insight into asynchronous
+  execution flow.
+
+ 
+The following tools are currently assigned to Tiers as follows
```

Comment:
Colon in the end?

Reply:
Tautology: the word "follow" is used here twice:

> The **following** tools are ... as **follows*

Reply:
Fixed